test/integration/actions-test.js

```diff
@@ -0,0 +1,186 @@
+'use strict';
+
+/* Tell jshint about mocha globals, and  */
+/* globals it */
+
+const {server, chai, mockAdapter} = require('../common');
+
+var Constants = require('../../constants');
+const WebSocket = require('ws');
+
+it('removes all existing actions', done => {
```

Comment:
In the end I do want all tests to be promise based (async functions)... It's too easy to forget done (and of course way less boilerplate like below)